backend/app/services/encryption.py

```
@@ -0,0 +1,119 @@
+
+    Attempts decryption — if it succeeds, the value is encrypted.
+    """
+    if not value:
+        return False
+    try:
+        k = key or get_encryption_key()
+        if not k:
+            return False
+        f = Fernet(k.encode() if isinstance(k, str) else k)
+        f.decrypt(value.encode())
+        return True
```

Ruff (0.15.4)

[warning] Consider moving this statement to an `else` block

(TRY300)